build_image.py

```diff
@@ -107,6 +107,7 @@ def main():
         # For secure build arguments
         if config.extra_build_args:
             base_command.extend(config.extra_build_args)
+        env = dict(os.environ, DOCKER_BUILDKIT="1")
```

Comment:
Why does this need to happen? Isn't line 79 sufficient?

Reply:
Good catch, this dates from when 79 did not exist. Removed it.